src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,61 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private final Connection connection;
+
+    private static ConnectionService INSTANCE = null;
+
+    private static final String DRIVER_POSTGRESQL = "org.postgresql.Driver";
+
+    private static final String JDBC_URL = "jdbc:postgresql://developers-db:5432/";
+
+    public ConnectionService() {
+        this.connection = connectToDb();
+    }
+
+    public Connection getConnection() {
+        return connection;
+    }
+
+    public Connection connectToDb() {
+        Connection connection = null;
+
+
+        try {
+
+            Class.forName(DRIVER_POSTGRESQL);
+            connection = DriverManager.getConnection(creationDatabaseUrl(), DATABASE_NAME, DATABASE_PASSWORD);
+            if (connection != null) {
+                System.out.println("Connection Established!");
+            } else {
+                System.out.println("Connection Failed!");
+            }
+
+
+        } catch (Exception e) {
+            System.out.println(e);
+        }
+        return connection;
+    }
+
+    public String creationDatabaseUrl() {
+        String formatUrl = "%s%s";
+        String url = String.format(formatUrl, JDBC_URL, DATABASE_NAME);
+        return url;
```

Comment:
Do not create an extra variable. Directly return the result of the execution.

Reply:
fix